src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraVersion.kt

```diff
@@ -0,0 +1,8 @@
+package no.skatteetaten.aurora.boober.model
+
+object AuroraVersion {
+
+    fun isFullAuroraVersion(tag: String): Boolean {
+        return Regex("^.*-b(.*)-([a-z]*)-(.*)$").matches(tag)
```

Comment:
Burde vi godta mer variasjon i navnet på _base image_? Kan det være at a-z er for snevert?